python/ray/serve/scripts.py

```diff
@@ -511,18 +511,18 @@ def config(address: str, name: Optional[str]):
                     sort_keys=False,
                 )
                 for app in serve_details.applications.values()
+                if app.deployed_app_config is not None
             ),
             end="",
         )
     # Fetch a specific app config by name.
     else:
-        if name not in serve_details.applications:
-            config = ServeApplicationSchema.get_empty_schema_dict()
+        app = serve_details.applications.get(name)
+        if app is None or app.deployed_app_config is None:
+            print(f'No config has been deployed for application "{name}".')
```

Comment:
what will `name` be if I don't specify a name?

Reply:
If it's single-app, then name=default. If it's multi app it will branch to different code which displays all non-null configs that are fetched.